src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -68,7 +74,13 @@ public LibLogScopeEventSubscriber(IScopeManager scopeManager)
 
         public void StackOnSpanOpened(object sender, SpanEventArgs spanEventArgs)
         {
-            SetCorrelationIdentifierContext(spanEventArgs.Span.TraceId, spanEventArgs.Span.SpanId);
+            var span = spanEventArgs.Span;
+            SetCorrelationIdentifierContext(
+                span.ServiceName,
```

Comment:
this should be the globally configured service name.

we want to be sure we associate the service name of the app, not the span.

Here is the code from Python:

https://github.com/DataDog/dd-trace-py/blob/0d01b297a9988204b4eb13af15a95ec940f0e1e0/ddtrace/contrib/logging/patch.py#L30-L53

Reply:
Same with version and env.

Reply:
Sounds good, will fix

Reply:
Fixed in 0d41db3f74c6f4d317c7a6fee3924920b35d83d1